toolkit/tools/imagecustomizerapi/pxe.go

```diff
@@ -13,11 +13,12 @@ var PxeIsoDownloadProtocols = []string{"ftp://", "http://", "https://", "nfs://"
 
 // Iso defines how the generated iso media should be configured.
 type Pxe struct {
-	KernelCommandLine KernelCommandLine  `yaml:"kernelCommandLine" json:"kernelCommandLine,omitempty"`
-	AdditionalFiles   AdditionalFileList `yaml:"additionalFiles" json:"additionalFiles,omitempty"`
-	InitramfsType     InitramfsImageType `yaml:"initramfsType" json:"initramfsType,omitempty"`
-	BootstrapBaseUrl  string             `yaml:"bootstrapBaseUrl" json:"bootstrapBaseUrl,omitempty"`
-	BootstrapFileUrl  string             `yaml:"bootstrapFileUrl" json:"bootstrapFileUrl,omitempty"`
+	KernelCommandLine KernelCommandLine   `yaml:"kernelCommandLine" json:"kernelCommandLine,omitempty"`
+	AdditionalFiles   AdditionalFileList  `yaml:"additionalFiles" json:"additionalFiles,omitempty"`
+	InitramfsType     InitramfsImageType  `yaml:"initramfsType" json:"initramfsType,omitempty"`
+	BootstrapBaseUrl  string              `yaml:"bootstrapBaseUrl" json:"bootstrapBaseUrl,omitempty"`
+	BootstrapFileUrl  string              `yaml:"bootstrapFileUrl" json:"bootstrapFileUrl,omitempty"`
+	KdumpBootFiles    *KdumpBootFilesType `yaml:"kdumpBootFiles" json:"kdumpBootFiles,omitempty"`
```

Comment:
Instead of using a pointer, I would just support the empty string in `KdumpBootFilesType`. And treat `""` and `"none"` the same way. This will probably help simplify the code.

Reply:
Talked with Chris offline - we're keeping the pointer to differentiate between the case where the user sets it to an empty string and the case where the user does not set it at all.